bot/exts/help_channels/_cog.py

```diff
@@ -535,3 +540,94 @@ async def update_available_help_channels(self) -> None:
+        """
+        if await _caches.claimants.get(message.channel.id) == message.author.id:
+            return  # Ignore messages sent by claimants
+
+        if not await _caches.help_dm.get(message.author.id):
+            return  # Ignore message if user is opted out of help dms
+
+        if message.content == f"{self.bot.command_prefix}close":
+            return  # Ignore messages that are closing the channel
+
+        session_participants = self._deserialise_session_participants(
+            await _caches.session_participants.get(message.channel.id) or ""
+        )
+
+        if message.author.id not in session_participants:
+            session_participants.add(message.author.id)
+
+            user = self.bot.get_user(message.author.id)
+
+            embed = discord.Embed(
+                title="Currently Helping",
+                description=f"You're currently helping in <#{message.channel.id}>",
+                color=discord.Colour.green(),
+                timestamp=message.created_at
+            )
+            embed.add_field(name="Conversation", value=f"[Jump to message]({message.message.jump_url})")
+            await user.send(embed=embed)
+
+            await _caches.session_participants.set(
+                message.channel.id,
+                self._serialise_session_participants(session_participants)
+            )
+
+    @commands.group(name="helpdm")
+    async def help_dm_group(self, ctx: commands.Context) -> None:
+        """
+        User will receive a embed when they are "helping" in a help channel.
+
+        If they have helpdms off they will won't receive an embed.
+        If they have helpdms on they will receive an embed.
+        """
+        if ctx.invoked_subcommand is None:
+            await ctx.send_help(ctx.command)
+
+    @help_dm_group.command(name="on")
+    async def on_command(self, ctx: commands.Context) -> None:
+        """Turns help dms on so the user will receive the participating dm."""
+        if await _caches.help_dm.get(ctx.author.id):
+            await ctx.send(f"{constants.Emojis.cross_mark}{ctx.author.mention} Help DMs are already ON!")
+
+            log.trace(f"{ctx.author.id} Attempted to turn Help DMs on but they are already ON")
+            return
+
+        await _caches.help_dm.set(ctx.author.id, True)
+
+        await ctx.send(f"{constants.Emojis.ok_hand} {ctx.author.mention} Help DMs ON!")
+
+        log.trace(f"{ctx.author.id} Help DMs OFF")
+
+    @help_dm_group.command(name="off")
+    async def off_command(self, ctx: commands.Context) -> None:
+        """Turns help dms off so the user wont receive the participating dm."""
+        if not await _caches.help_dm.get(ctx.author.id):
+            await ctx.send(f"{constants.Emojis.cross_mark} {ctx.author.mention} Help DMs are already OFF!")
+
+            log.trace(f"{ctx.author.id} Attempted to turn Help DMs on but they are already OFF")
+            return
+
+        await _caches.help_dm.set(ctx.author.id, False)
```

Comment:
Instead of overwriting the value with False, maybe we can just remove them from the dict entirely?

That way, this cache only includes those that have opted in